### PR TITLE
add README for planning

### PR DESCRIPTION
**Pull request by mmandlis**
fixes: https://github.com/PolymerLabs/arcs/issues/2405

### CODE REVIEW

File: src/runtime/plan/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+/server $ env {args} npm run server:start
+```
+Optionally prefix your command with the following optional environment arguments:
+```
+DEBUG=true
+ARCS_USER_ID=xxxxx
+STORAGE_KEY_BASE  (default is pouchdb://localhost:8080/user/)
+```
+Navigate to the Arcs page and use the storage, **plannerStorage** and **plannerOnlyConsumer** parameters:  
+https://localhost:8080/shells/web-shell?onlyConsumer=true&storage=pouchdb://localhost:8080/user/&plannerStorage=pouchdb://localhost:8080/user/
```
Comment (shaper) [inline src/runtime/plan/README.md]: The `onlyConsumer` and `storage` query params here don't match line 55, looks like they need to be updated (`storage` duplicative with `plannerStorage` at end).

File: src/runtime/plan/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
```
Comment (shaper) [inline src/runtime/plan/README.md]: Add a TODO to add instructions on how to bring up a Pouch server on a Personal Cloud server once we've gotten far enough to do so.

File: src/runtime/plan/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+To use PouchDb storage for suggestion go to:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerStorage=pouchdb://local/random-db-name  
+The suggestion values will be found in Chrome's inspector, under Application > Storage > Local Storage.
+
+
+To populate Arcs Extension strategy explorer, add the **plannerDebug** url param:  
+https://polymerlabs.github.io/arcs-live/shells/web-shell?plannerDebug
+
+# Remote Planning 
+To consume plans produced on the server, disable shell planning with an **onlyConsumer** url param, ie:
```
Comment (shaper) [inline src/runtime/plan/README.md]: `onlyConsumer` is cited here but the example below uses `plannerOnlyConsumer`, which is accurate?

File: src/runtime/plan/README.md (hunk 2 above)
Comment (shaper) [reply src/runtime/plan/README.md]: ...by which I mean, we'll probably link to them, not write them all out here. Perhaps linking to `server/README.md` is sufficient for now.

File: src/runtime/plan/README.md (hunk 1 above)
Comment (mmandlis) [reply src/runtime/plan/README.md]: done. 
(note to self: writing readme and renaming params in parallel, not a good idea!)

File: src/runtime/plan/README.md (hunk 2 above)
Comment (mmandlis) [reply src/runtime/plan/README.md]: added a link to server/readme

File: src/runtime/plan/README.md (hunk 3 above)
Comment (mmandlis) [reply src/runtime/plan/README.md]: fixed. i actually moved this part to the top sections with all the other links to arcs-live.

**Comment by mmandlis** [comment]
Addressed comments, PTAL

**Pull request merged**